vignettes/tidy-data.Rmd

```diff
@@ -137,15 +138,17 @@ This format is also used to record regularly spaced observations over time. For
 billboard
 ```
 
-To tidy this dataset, we first gather together all the `wk` columns. The column names give the `week` and the values are the `rank`s:
+To tidy this dataset, we first use pivot_longer to gether up all the `wk` columns. The column names give the `week` and the values are the `rank`s:
 
 ```{r}
 billboard2 <- billboard %>% 
-  gather(week, rank, wk1:wk76, na.rm = TRUE)
+  pivot_longer(cols = c(wk1:wk76), names_to = "week", values_to = "rank") %>% 
+  filter(!is.na(rank)) %>% 
+  arrange(week, artist)
 billboard2
 ```
 
-Here we use `na.rm` to drop any missing values from the gather columns. In this data, missing values represent weeks that the song wasn't in the charts, so can be safely dropped.
+Here we use `filter(!is.na())` to drop any missing values from the rank column. In this data, missing values represent weeks that the song wasn't in the charts, so can be safely dropped.
```

Comment:
Can you please use `values_drop_na = TRUE` instead?

Reply:
Sure. Done here and elsewhere.